docs/components/site.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
```

Comment:
Removed the Mapbox-hosted open sans font import from the page shell, but it's still being imported from Assembly, so this is probably unnecessary.